frameworks/Rust/axum/src/database_mongo.rs

```diff
@@ -14,14 +15,15 @@ use mongodb::Database;
 pub struct DatabaseConnection(pub Database);
 
 #[async_trait]
-impl<B> FromRequest<B> for DatabaseConnection
+impl<S, B> FromRequest<S, B> for DatabaseConnection
```

Comment:
All these `FromRequest` implementations should be `FromRequestParts`, because they don't care about the body of the request. Then you can also revert the change in extractor order for the handler functions.

Reply:
I have pushed the changes FromRequest ->  FromRequestParts.

Yeah, I have checked all included crates for the latest version and updated them.

In my opinion, the benchmarks should represent a performance figure which is representative at the time when the benchmark is executed. 
Meaning, that the latest packages should be used. 
This also gives the possibility to compare benchmark history and in cases of regression maybe find crates that don't have benchmarks in their CI.

There is theoretically a 1.0.0-RC.1 of hyper available, but the latest update was in October, and it dropped 
`hyper::server::conn::AddrIncoming` so there would be more refactoring necessary. 
But as there are updates on the 0.14.x version this should be fine.

 I tested on my local notebook and all tests pass.
 
 I highly appreciate your time and patience with a Rust noob.